CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,65 @@
+# Contributor Covenant Code of Conduct
```

Comment:
It's weird that it's recommended to add this file when the code of conduct seems to be "inherited" from the org https://github.com/ansys/openapi-common?tab=coc-ov-file#contributor-covenant-code-of-conduct.